### ISSUE TITLE
Manual Build Failed

### ISSUE THREAD

**Issue by BigFish2086** [PortSwigger/turbo-intruder#134]
I tried to build the master branch manually using ` ./gradew build fatjar ` however i got the following error
![image](https://github.com/PortSwigger/turbo-intruder/assets/63132227/8ad2b7da-497f-4f85-bff4-1b15568785f4)

with this configuration for java & gradle for this ` ubuntu:18.04 ` docker instance:
![image](https://github.com/PortSwigger/turbo-intruder/assets/63132227/a7e70424-2d45-4419-a258-bf025b951bd0)

It seems to me that the last version I can build is this commit: f30d741 i.e v1.32
and any other version after that can't be built manually

**Comment by albinowax** [comment]
Sorry about this, I believe you can make compilation work by editing the build.gradle and replacing `compileOnly files('montoya-api-2023.7-13-BLACKHAT.jar')` with `implementation net.portswigger.burp.extensions:montoya-api:2023.9`. 

I'll land a proper fix once I escape vegas.

**Comment by BigFish2086** [comment]
This line almost fixed the problem. I changed it and got the following error:
![turbo-needs-jvm-17](https://github.com/PortSwigger/turbo-intruder/assets/63132227/24f3092f-81d2-49eb-96dd-2a9aef7a9866)

I changed the lines at the start of the file ` build.gradle ` to the following:
```
sourceCompatibility = 17
targetCompatibility = 17
```

It worked now by changing also the JDK version from 11 to 17 but with these warnings
![src-17-target-17](https://github.com/PortSwigger/turbo-intruder/assets/63132227/75052572-878d-4c27-89a4-b973fe3502b1)

I actually wanted to try it with older versions of burp, but I think it's not compatible.
Anyways, thanks for your help. Enjoy yourself in vegas.
